lib/functions

```diff
@@ -118,6 +118,14 @@ function install_nodejs {
     npm install npm -g
 }
 
+# install_python() - Install Python 2.7 and other tools necessary for development.
+function install_python {
+    install_package python2.7
+    install_package python-dev
+    install_package python-pip
```

Comment:
I prefer to install the latest version

    curl -sL https://bootstrap.pypa.io/get-pip.py | python